src/org/opendatakit/briefcase/ui/pull/PullPanel.java

```diff
@@ -119,7 +119,7 @@ private void onSource(TransferPanelForm view, TransferForms forms, Source<?> sou
       updateActionButtons();
     }).onError(cause -> {
       log.warn("Unable to load form list from {}", source.getDescription(), cause);
-      errorMessage("Error Loading Forms", "We haven't been able to load forms using the saved source. Try reloading it or reset it, please.");
+      errorMessage("Error Loading Forms", "Briefcase hasn’t been able to load forms using the configured source. Try Reload or Reset.");
```

Comment:
Sorry I didn’t think of this earlier, but “hasn’t been able to” suggests repeated attempts. I suggest “Briefcase wasn’t able to ...”.